etc/cvmfs/common.conf

```diff
@@ -2,6 +2,6 @@
 # cvmfs does not parse included files for variable names, so first need
 #   to set in the domain config any variable set here.
 
-if [ -n "$CVMFS_HTTP_PROXY" ] && [ "$CVMFS_HTTP_PROXY" != "DIRECT" ] && [ "$CVMFS_HTTP_PROXY" != "auto;DIRECT" ]; then
+if [ -n "$CVMFS_HTTP_PROXY" ] && [ "$CVMFS_HTTP_PROXY" != "DIRECT" ]; then
```

Comment:
This check for "auto;DIRECT" is important, because the WPAD server inserts the backup proxies when they are needed.